test/al_aws_collector_test.js

```diff
@@ -254,6 +254,32 @@ describe('al_aws_collector tests', function() {
             });
         });
 
+        it('checkin success registered with env vars not set', function (done) {
+            delete process.env.ingest_api;
```

Comment:
maybe set these to undefined, saving the previous values, and resetting them before the test finishes. I think this will screw up later tests if they decide to use those variables.